mapdata/WarcraftLegacies/AbilityData/Skin/1378955585.json

```diff
@@ -58,7 +58,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Gives nearby Sentinel units 30% movement speed increase."
+      "Value": "Gives nearby Sentinel units \u003CA11R,DataA1,%\u003E% movement speed increase."
```

Comment:
Need to update the description of the Power too, it currently still says 24%: 
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/27d57d25-3cd5-4b47-abcc-326e118bff7a)

You can find it in line 180 of the Sentinels.cs file.